EducationPortal/EducationPortal.BLL/Response/RegisterResponse.cs

```diff
@@ -1,7 +0,0 @@
-﻿namespace EducationPortal.BLL.Response
```

Comment:
Is this an empty file?

Reply:
it happened by accident. I've already removed it